aws/cloudformation/standalone/gen_ai_curriculum/gen_ai_curriculum.yml.erb

```diff
@@ -0,0 +1,82 @@
+# Gen AI Curriculum Resources
+# This template is manually deployed to create/update Gen AI Curriculum resources.
+# TODO:
+  # Fix hardcoded execution role ARN
+
+AWSTemplateFormatVersion: 2010-09-09
+
+Description: SageMaker resources for Gen AI Curriculum.
+
+Parameters:
+  Environment:
+    Type: String
+    Default: "test"
+    Description: "Environment being deployed to (prod or test)"
+  HuggingFaceToken:
+    Type: String
+    Description: "HuggingFace token required for deploying HuggingFace models."
+  HuggingFaceImageUri:
+    Type: String
+    Default: "763104351884.dkr.ecr.us-east-1.amazonaws.com/huggingface-pytorch-tgi-inference:2.3.0-tgi2.2.0-gpu-py310-cu121-ubuntu22.04-v2.0"
+    Description: "HuggingFace Image URI to use for the model."
+Resources:
+<% Config::ENDPOINT_CONFIGS.each do |config| %>
+  <%= config[:model_name] %>Model:
+    Type: AWS::SageMaker::Model
+    Properties:
+      ModelName: !Sub "<%= "#{config[:model_name]}-${Environment}-#{Time.now.to_i}" %>"
+      ExecutionRoleArn: !Sub "arn:aws:iam::${AWS::AccountId}:role/AmazonSageMaker-ExecutionRole-test"
```

Comment:
@code-dot-org/infrastructure I manually created this role ("AmazonSageMaker-ExecutionRole-test") in the dev account for testing. We'll need to make a similar execution role with SageMaker permissions in production; should this also be created via Cloudformation?

Reply:
Lets define the role in this template, then we can `!Ref` it by name.

Something like:

```
  SagemakerFullAccessRole:
    Type: 'AWS::IAM::Role'
    Properties:
      AssumeRolePolicyDocument:
        Version: '2012-10-17'
        Statement:
          - Effect: Allow
            Principal:
              Service: sagemaker.amazonaws.com
            Action: 'sts:AssumeRole'
      ManagedPolicyArns:
        - 'arn:aws:iam::aws:policy/AmazonSageMakerFullAccess'
```

Reply:
After adding this I get 
```
An error occurred (InsufficientCapabilitiesException) when calling the CreateChangeSet operation: Requires capabilities : [CAPABILITY_IAM]
``` 
when trying to deploy the stack. Would we need admin privileges to do this?

Reply:
Once we deploy #61001 we can update this to be an !ImportValue